renormalizer/mps/matrix.py

```diff
@@ -7,8 +7,7 @@
 
 import numpy as np
 
-from renormalizer.mps.backend import xp, backend, cp
-
+from renormalizer.mps.backend import np, backend, xp, USE_GPU, xp as xp
```

Comment:
???

Reply:
I think this is a bug of the refactoring tool of PyCharm...